### PR TITLE
Proposal: IP Address CIDR Request

### PR DESCRIPTION
**Pull request by ronaldheft-usds**
This is a proposal for the IP CIDR request we need to submit to the CDC, which is a prerequisite to peering with the Palo Alto firewall. Items to discuss are listed within the proposal.

### CODE REVIEW

**Review by jimduff-usds — COMMENTED** [review]
I'd just say that its highly likely we'll stay with a serverless architecture, at least for those components which might need to support very heavy usage.   This would imply that 512 is on the high end of our need for IPs.

File: prime-router/docs/proposals/0008-cdc-ip-space.md
Comment (ronaldheft-usds) [inline prime-router/docs/proposals/0008-cdc-ip-space.md:125]: @jimduff-usds That's a fair statement that we're requesting too many IPs. Thoughts on 128 IPs per VNET?

```suggestion
**x.x.x.x/25**

### Why a `/25`?

* A `/25` will grant us 128 IP addresses per VNET
* While we currently use a limited number of IP addresses, we cannot anticipate our future needs
* We may want to utilize private endpoints again in the future
* While our current utilized resources do not consume VNET ip addresses when scaled (ex. function app, database, etc), we may in the future have bare containers that need to be scaled, which will consume an IP per server

### Why request a `/25` in each environment-region pair?

* Environments should be identical
    * When we conduct full-scale load testing in a lower environment, we’ll want the same number of IPs to verify our scaling logic
* Regions should allow for full redundant replication
    * While the only resource currently deployed in our secondary region is a database replica, we’ll want the capability to stand up our entire system in our secondary region
    * This would grant us full region redundancy and/or provide a closer network path for users on the west coast

#### Subnet Usage

| Subnet | Description | CIDR | IPs | Used | Available |
|:--|:--|--:|--:|--:|--:|
| public | Internet-routable resources | x.x.x.x/28 | 16 | 5 + 0 = 5 | 11 |
| container | Docker containers (ex. SFTP test) | x.x.x.x/28 | 16 | 5 + 2 = 7 | 9 |
| private | VNET-only; not Internet-routable | x.x.x.x/28 | 16 | 5 + 0 = 5 | 11 |
| GatewaySubnet | Not used under a CDC VNET |  | |  |  |
| endpoint | Not used under a CDC VNET |  | |  |  |

#### VNET Summary

| Count | Label |
|--:|:--|
| 128 | IPs in CIDR |
| 48 | Allocated IPs to Subnets |
| 80 | Available IPs |
```

**Comment by ronaldheft-usds** [comment]
Discussed during our dev sync today. Could I get an approval from someone before merging? Thanks!

**Review by td-usds — APPROVED** [review]

**Pull request merged**
